test/language/expressions/unary-minus/bigint.js

```diff
@@ -0,0 +1,34 @@
+// Copyright (C) 2017 Josh Wolfe. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+description: Unary minus for BigInt values
+esid: sec-numeric-types-bigint-unaryMinus
+info: |
+  BigInt::unaryMinus (x)
+
+  The abstract operation BigInt::unaryMinus with an argument x of BigInt type returns the result of negating x.
+
+  Note: There is only one 0n value; -0n is the same as 0n.
+
+features: [BigInt]
+---*/
+
+assert.sameValue(-0n, 0n, "-0n === 0n");
```

Comment:
I really like the info with the reminder `There is only one 0n value; -0n is the same as 0n.`. 

Nice test. 👍